lib/zendesk_apps_support/validations/manifest.rb

```diff
@@ -272,6 +272,7 @@ def invalid_v1_location(package)
 
         def invalid_location_uri_error(package, path)
           return nil if path == ZendeskAppsSupport::Manifest::LEGACY_URI_STUB
+          return nil if path.index('{{setting.') != nil
```

Comment:
Or if you want to write it shorter:
`return nil if path == ZendeskAppsSupport::Manifest::LEGACY_URI_STUB || path.index('{{setting.') != nil`